assets/wire/utils.ts

```diff
@@ -8,19 +8,21 @@ export const DISPLAY_ABSTRACT = getConfig('display_abstract');
 
 const STATUS_KILLED = 'canceled';
 
-
+const DEFAULT_NO_VERSION = 0;
 /**
  * Returns the item version as integer
  *
  * @param {Object} item
  * @returns {number}
  */
-export function getIntVersion(item: IArticle | IAgendaItem | undefined): number | undefined {
+export function getIntVersion(item: IArticle | IAgendaItem | undefined): number {
```

Comment:
This is hacky as well. It was returning a number or undefined which makes sense. Now you're doing a unnecessary breaking change where it's not possible to distinguish value zero from no value.

A better approach would be to change it here
```
const version = getIntVersion(this.props.itemsById[itemId]);

isRead={version == null || version === this.props.readItems[itemId]}
```